include/rdma/fabric.h

```diff
@@ -546,6 +546,7 @@ enum fi_type {
 	FI_TYPE_CQ_EVENT_FLAGS,
 	FI_TYPE_MR_MODE,
 	FI_TYPE_OP_TYPE,
+	FI_TYPE_SIZE_T,
```

Comment:
maybe FI_TYPE_NUMERIC_FIELD will be okay? I just want to define common `fi_type` that can be used for numeric value like `max_msg_size` and etc (all of them has `size_t` datatype).